osbuild/__init__.py

```diff
@@ -98,13 +137,13 @@ def run(self, argv, binds=None, readonly_binds=None, **kwargs):
 
         Its arguments mean the same as those for subprocess.run().
         """
+
         return subprocess.run([
             "systemd-nspawn",
             "--quiet",
             "--register=no",
             "--as-pid2",
             "--link-journal=no",
-            "--volatile=yes",
```

Comment:
I thought we have to keep this, because it also sets up `/var` and `/tmp`?

Reply:
We can't use --volatile, I think nspawn will set up "all the missing bits" (according to Lennart), but I'll double check.

Reply:
Yup, this seems to be working correctly (as promised).